libs/langchain-community/src/vectorstores/pgvector.ts

```diff
@@ -527,9 +535,8 @@ export class PGVectorStore extends VectorStore {
 
     const queryString = `
       DELETE FROM ${this.computedTableName}
-      WHERE ${collectionId ? "collection_id = $2 AND " : ""}${
-      this.idColumnName
-    } = ANY($1::uuid[])
+      WHERE ${collectionId ? "collection_id = $2 AND " : ""}${this.idColumnName
```

Comment:
Can you run `yarn format`? Seems like it's blocking CI

I don't have permission to merge to your branch

Reply:
Sorry about that! I ran it on the wrong branch before 🙈